osx port

axcap opened the pull request.

ozkl commented:
Thanks for the port. I'd like to merge this but I think it is better to name it SDL port instead of OS X port since it does not use OS X specific code. Also let's remove binaries. What do you think?

axcap commented:
You are absolutely right. To the best of my knowledge I changed all 'osx'-references to 'sdl' and removed 2 binary files (main executable and wad-resource). Should be better now.
